openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -2190,9 +2220,11 @@ The definitions in (#authorization-ldp-vc) apply for Credentials of this type as
 
 The definitions in (#credential-response-jwt-vc-json) apply for Credentials of this type as well.
 
-## ISO mDL
+## Mobile Documents or mdocs (ISO/IEC 18013 and ISO/IEC 23220 series) {#mdocs}
+
+This section defines a Credential Format Profile for Credentials complying with the mobile document (mdoc) format defined in [@!ISO.18013-5] and the ISO/IEC 23220 series.
 
-This section defines a Credential Format Profile for Credentials complying with [@!ISO.18013-5].
+ISO/IEC 18013-5:2021 [@ISO.18013-5] defines a mobile driving license (mDL) Credential in the mobile document (mdoc) format. Although ISO/IEC 18013-5:2021 [@ISO.18013-5] is specific to mobile driving licenses (mDLs), the Credential format can be utilized with any type of Credential (or mdoc document types). The ISO/IEC 23220 series has extracted components from ISO/IEC 18013-5:2021 [@ISO.18013-5] that are common across document types to facilitate the profiling of the specification for other document types. The core data structures are shared between ISO/IEC 18013-5:2021 [@ISO.18013-5] and technical specifications of the ISO/IEC 23220 series such as: ISO/IEC 23220-2 [@ISO.23220-2], ISO/IEC 23220-3 [@ISO.23220-3] and ISO/IEC 23220-4 [@ISO.23220-4] which are encoded in CBOR and secured using `COSE_Sign1`. Specifically, ISO/IEC 23220-3 [@ISO.23220-3] defines data structures of building blocks for the issuance of mdocs.
```

Comment:
```suggestion
ISO/IEC 18013-5:2021 [@ISO.18013-5] defines a mobile driving license (mDL) Credential in the mobile document (mdoc) format. Although ISO/IEC 18013-5:2021 [@ISO.18013-5] is specific to mobile driving licenses (mDLs), the Credential format can be utilized with any type of Credential (or mdoc document types).
```
could we keep information to the minimum..?

Reply:
@Sakurann I simplified the PR, only having 23220-1 as a informative reference. I thought it would make sense to keep that at least. Please review again.